smartctl.go

```diff
@@ -435,3 +436,60 @@ func (smart *SMARTctl) mineDeviceERC() {
 		)
 	}
 }
+
+func (smart *SMARTctl) mineSCSIGrownDefectList() {
+	scsi_grown_defect_list := smart.json.Get("scsi_grown_defect_list")
+	if scsi_grown_defect_list.Exists() {
+		smart.ch <- prometheus.MustNewConstMetric(
+			metricSCSIGrownDefectList,
+			prometheus.CounterValue,
+			scsi_grown_defect_list.Float(),
+			smart.device.device,
+			smart.device.family,
+			smart.device.model,
+			smart.device.serial,
+		)
+	}
+}
+
+func (smart *SMARTctl) mineSCSIErrorCounterLog() {
+	SCSIHealth := smart.json.Get("scsi_error_counter_log")
+	if SCSIHealth.Exists() {
+		smart.ch <- prometheus.MustNewConstMetric(
+			metricReadErrorsCorrectedByRereadsRewrites,
+			prometheus.CounterValue,
+			SCSIHealth.Get("read.errors_corrected_by_rereads_rewrites").Float(),
+			smart.device.device,
+			smart.device.family,
+			smart.device.model,
+			smart.device.serial,
+		)
+		smart.ch <- prometheus.MustNewConstMetric(
+			metricReadTotalUncorrectedErrors,
+			prometheus.CounterValue,
```

Comment:
This either needs to be a `GaugeValue`, or the metric name needs to end in `_total`.

Same goes for all the other metrics.